src/DynamoSandbox/Program.cs

```diff
@@ -229,21 +230,43 @@ private static void PreloadShapeManager(ref string geometryFactoryPath, ref stri
             preloaderLocation = preloader.PreloaderLocation;
         }
 
+        [DllImport("msvcrt.dll")]
+        public static extern int _putenv(string env);
+
         [STAThread]
         public static void Main(string[] args)
         {
             DynamoViewModel viewModel = null;
+            var supportedLocale =
+            new HashSet<string>(new[]
+                {
+                    "cs-CZ", "de-DE", "en-US", "es-ES", "fr-FR", "it-IT",
+                    "ja-JP", "ko-KR", "pl-PL", "pt-BR", "ru-RU", "zh-CN", "zh-TW"
+                });
```

Comment:
This declaration for `supportedLocale` can be moved into the `else` condition so that it won't be created unnecessarily.